### PR TITLE
Use 5" inch layout on small mobile screens (wasm)

### LINKED ISSUE #919: Use smaller 5" layout variant on phones (wasm)
Current WASM build uses the larger 7" display 1024:600 layout variant. This is a good choice for large desktop and tablet screens, but the smaller 5" variant might be more appropriate with bigger text and buttons when accessing the website from small phone screens.

### PR DESCRIPTION
**Pull request by jpetrell**
Fixes #919.

### CODE REVIEW

File: src/theme.cpp [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+#include "theme.h"
+
+using namespace Victron::VenusOS;
+
+#if defined(VENUS_WEBASSEMBLY_BUILD)
+#include <emscripten.h>
+
+EM_JS(int, getScreenWidth, (), {
+  return Math.min(screen.width, screen.height);
+});
```
Comment (jpetrell) [inline src/theme.cpp:15]: Cannot declare EM_JS macro on the header as that will result in duplicate symbols error
as any place that includes the header will have own version of the same global function.

File: src/theme.cpp [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+#endif
+
+Theme::Theme(QObject *parent) : QObject(parent)
+{
+#if defined(VENUS_WEBASSEMBLY_BUILD)
+	setScreenSize(getScreenWidth() >= 480
+				  ? Victron::VenusOS::Theme::SevenInch
+				  : Victron::VenusOS::Theme::FiveInch);
```
Comment (jpetrell) [inline src/theme.cpp:24]: Smartphones have 320 - 480 CSS independent pixel wide screens.

Comment (blammit) [reply src/theme.cpp:24]: Could you add this as a source comment please?

Comment (jpetrell) [reply src/theme.cpp:25]: Good idea. Done. :+1:

**Review by chriadam — APPROVED** [review]

**Pull request merged**